src/validators/RequestSchema.js

```diff
@@ -63,7 +63,143 @@ const UpdateHomepageSchema = Joi.object({
       permalink: Joi.string().required(),
       notification: Joi.string().allow(""),
       image: Joi.string(),
-      sections: Joi.array().required(),
+      sections: Joi.array()
+        .items(
+          // Hero section
+          Joi.object({
+            hero: Joi.object({
+              variant: Joi.string().allow(
+                "side",
+                "image",
+                "floating",
+                "center",
+                ""
+              ),
+              backgroundColor: Joi.string().allow("black", "white", "gray", ""),
+              background: Joi.string().allow(""),
+              size: Joi.string().allow("sm", "md", ""),
+              alignment: Joi.string().allow("left", "right", ""),
```

Comment:
This is from a static analysis of the [template codebase](https://github.com/isomerpages/isomerpages-template/blob/next-gen/_includes/homepage/hero/hero-floating-layout.html#L13) that checks for `right` or otherwise, and the frontend CMS that [only provides for these two options](https://github.com/isomerpages/isomercms-frontend/blob/develop/src/types/hero.ts#L3).